Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/modules/detectiontracker/DefaultExposureDetectionTracker.kt

```diff
@@ -134,6 +117,54 @@ class DefaultExposureDetectionTracker @Inject constructor(
         }
     }
 
+    private fun finishExposureDetectionWithoutIdentifier(
```

Comment:
Would an extension function improve readability here? 

Something like `fun Map<String, TrackedExposureDetection>.findOrCreateByIdentifier(identifier: String?)`

Reply:
I changed it to:
private fun Map<String, TrackedExposureDetection>.findUnfinishedOrCreateIdentifier(): String {